gremlin-core/src/main/java/org/apache/tinkerpop/gremlin/structure/util/star/StarGraph.java

```diff
@@ -869,6 +870,32 @@ public Vertex inVertex() {
         }
     }
 
+    public final class StarEdgeId {
```

Comment:
This is a bit worrisome. Does this mean that if a user accesses an edge identifier in OLAP they won't get back their edge id, they will instead get a `StarEdgeId`?